packages/core/tests/test.ts

```diff
@@ -0,0 +1,28 @@
+import { expect, test } from '@playwright/test'
+
+/**
+ * This test will make sure that the `<canvas>` element created by `<Canvas>` is
+ * resized according to its parent element.
+ */
+test('basics/viewport-resize', async ({ page }) => {
+  await page.setViewportSize({
+    width: 600,
+    height: 600
+  })
+  await page.waitForTimeout(500)
```

Comment:
Any way to await something that's event-driven? Seems like maybe a potential source of test flakiness